src/ca/noxid/lab/messages.properties

```diff
@@ -388,3 +388,6 @@ TscPane.34=empty
 TscPane.43=Command:
 TscPane.45=Description:\n
 TscPane.9=Command:
+MapDeleteConfirm.0=Are you sure you want to delete map\n"
+MapDeleteConfirm.1="?
```

Comment:
There is a confirm dialog for deleting mapdata using the delete key at EditorApp.java line 649; you should reuse the localization strings from that one instead of making new ones.

As well, I don't think punctuation by itself necessarily needs its own localization entry but maybe that's just me.